core/src/test/java/com/linecorp/armeria/client/endpoint/WeightedRoundRobinStrategyTest.java

```diff
@@ -55,14 +56,14 @@ public void setup() {
 
     @Test
     public void select() {
-        assertThat(EndpointGroupRegistry.selectNode(ctx, "endpoint")).isNotNull();
+        assertThat(selectNode(ctx, "endpoint")).isNotNull();
 
-        assertThat(catchThrowable(() -> EndpointGroupRegistry.selectNode(ctx, "empty")))
+        assertThat(catchThrowable(() -> selectNode(ctx, "empty")))
                 .isInstanceOf(EndpointGroupException.class);
     }
 
     @Test
-    public void testRoundRobinSelect() {
+    public void testRoundRobinSelect() throws Exception {
```

Comment:
You don't need to add `throws Exception` by using `.join()` instead of `.get()`

Reply:
Fixed